assets/javascript/index.js

```diff
@@ -39,8 +42,22 @@ paymentForm.addEventListener("submit", (e) => {
     hideElements()
     renderRatingModal()
     renderMessage()
+    clearForm()
 })
 
+// closes payment modal if user wants to edit order
+closeModalBtn.addEventListener("click", (e) => {
+    paymentModal.classList.add("hide")
+})
+
+// clears input values
+function clearForm() {
```

Comment:
Is there a better way to do this? 

How come the default behavior doesn't clear the form after submission? I thought it did, based off the Scrims? Although, when I tried it, this technique didn't work for me.

Reply:
When we submit a form, it automatically refreshes the page. We then use e.preventDefault within the paymentForm event listener, which allows us to stop refreshes and allows us to render the rating modal. So since we are refreshed it we have to clear the inputs manually. I think this is the best way unless there is an alternative.

Reply:
Okay. I was just wondering.